support/hooks/prepare-commit-msg

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+#
+# A git commit hook that will automatically append a DCO signoff to the bottom
```

Comment:
Could we just enforce that a DCO is present? This would allow flows where the user automatically appends DCO (which I have already) but ensures that there are no mistakes made (trust but verify).

Reply:
(Handled in the discussion below.)